pom.xml

```diff
@@ -32,6 +32,31 @@
     </dependencies>
   </dependencyManagement>
 
+  <repositories>
```

Comment:
I think it should be enough to have the repositories listed in the parent pom project